Doc/tutorial/classes.rst

```diff
@@ -737,19 +737,23 @@ to code that is byte-compiled together.  The same restriction applies to
 Odds and Ends
 =============
 
-Sometimes it is useful to have a data type similar to the Pascal "record" or C
-"struct", bundling together a few named data items.  An empty class definition
-will do nicely::
+"struct", bundling together a few named data items. The idiomatic approach
+is to use :mod:`dataclasses` for this purpose::
 
-   class Employee:
-       pass
+    from dataclasses import dataclasses
 
-   john = Employee()  # Create an empty employee record
+    @dataclass
+    class Employee:
+        name: str
+        dept: str
+        salary: int
 
-   # Fill the fields of the record
-   john.name = 'John Doe'
-   john.dept = 'computer lab'
-   john.salary = 1000
+    john = Employee("john", "computer lab", 1000)
+
```

Comment:
I think it makes more sense for `john` to be defined as part of the REPL example rather than the module example. Also, by adding the `::` here, we can cleanly separate the interactive REPL example from the module example just above it, similar to how it is done with the `Reverse` class on line 834 further down in this same file:

![image](https://user-images.githubusercontent.com/66076021/209440114-90ea6183-c03a-41a1-a399-8a15a2a98e71.png)

```suggestion
::

    >>> john = Employee("john", "computer lab", 1000)
```